Resources/config/orm.xml

```diff
@@ -95,7 +95,7 @@
 
         <service id="doctrine.orm.configuration" class="%doctrine.orm.configuration.class%" abstract="true" public="false" />
 
-        <service id="doctrine.orm.entity_manager.abstract" class="%doctrine.orm.entity_manager.class%" abstract="true" />
+        <service id="doctrine.orm.entity_manager.abstract" class="%doctrine.orm.entity_manager.class%" abstract="true" lazy="true" />
```

Comment:
I agree with @nicolas-grekas and @Ocramius. there is no need to make it conditional as it is already handled gracefully.